src/csrc/rl/utils.cpp

```diff
@@ -66,9 +66,10 @@ void init_parameters(std::shared_ptr<ModelWrapper> model) {
     auto dim = val.dim();
 
     if (key.find("weight") != std::string::npos) {
-      // likely a conv or linear, we can use kaiming
+      // likely a conv or linear, we can use ortho
       if (dim >= 2) {
-        torch::nn::init::kaiming_uniform_(val);
+        //torch::nn::init::kaiming_uniform_(val);
```

Comment:
Is kaiming initialization never used now?

Reply:
So I learned that for most of the RL stuff you want to use orthogonal initialization. Maybe one can add a parameter to switch? Not sure how important that is but stable baselines basically uses ortho almost everywhere

Reply:
I have removed the commented aiming lines.